app/controllers/home_controller.rb

```diff
@@ -19,15 +19,17 @@ def dashboard
       user_id: current_user.id,
       recent: true,
       include_private: true,
-      include_adult: !current_user.sfw_filter?
+      include_adult: !current_user.sfw_filter?,
```

Comment:
If we were learning from PHP's style we'd never get anything done! I say we just use the bbatsov style guide default and call it good, so we don't have to [bikeshed](http://bikeshed.org)